.eslintrc

```diff
@@ -2,7 +2,14 @@
     "extends": ["standard", "eslint:recommended"],
     "rules": {
         "indent": ["error", 4],
-        "no-shadow": ["error"]
+        "no-shadow": ["error"],
+        "no-restricted-syntax": [
```

Comment:
Very cool, but please can you split this out as a separate change. It's unrelated to this PR. That way we can also change "warn" to "error" and fix all the failures at the same time.